File: functions/cpu_intensive_1.js
exports.handler = async (event) => {
    mySlowFunction(8);
    const response = {
        statusCode: 200,
        body: JSON.stringify('test'),
    };
    return response;
};


function mySlowFunction(baseNumber) {
	console.time('mySlowFunction');
	let result = 0;
	for (var i = Math.pow(baseNumber, 7); i >= 0; i--) {
		result += Math.atan(i) * Math.tan(i);
	}
	console.timeEnd('mySlowFunction');
}


// Adapted from: https://gist.github.com/sqren/5083d73f184acae0c5b7

File: functions/cpu_intensive_1.py
import json
from datetime import datetime
import math

def lambda_handler(event, context):
    cpu_intensive(8)
    return {
        'statusCode': 200,
        'body': json.dumps('works')
    }

def cpu_intensive(number):
    start = datetime.now()

    result = 0
    for x in range(number ** 7):
        result += math.tan(x) * math.atan(x)

    end = datetime.now()
    print("Duration =", end - start)

    return result
